package.diff

```diff
@@ -1,18 +1,16 @@
---- package-2.json	2021-11-04 19:31:35.170000000 +0000
-+++ package.json	2021-11-04 19:33:42.300734406 +0000
-@@ -51,6 +51,8 @@
-     "js-sha256": "^0.9.0",
-     "lodash.throttle": "4.1.1",
-     "lol-lockfile-parser": "^1.0.2",
-+    "@electron/remote": "^2.0.1",
-+    "electron-prompt": "^1.7.0",
-     "make-promises-safe": "^5.1.0",
-     "node-fetch": "^2.6.0",
-     "plist": "^3.0.1",
-@@ -69,4 +71,4 @@
-     "winston-daily-rotate-file": "^4.5.5",
-     "ws": "^7.2.0"
-   }
--}
-\ No newline at end of file
-+}
+--- package-2.json	2022-12-06 21:34:33.277584409 -0300
++++ package.json	2022-12-06 21:30:42.089629640 -0300
+@@ -16,12 +16,13 @@
+     "@fontsource/oswald": "^4.5.1",
+     "@fontsource/roboto": "^4.5.1",
+     "@google-cloud/storage": "^5.8.3",
+     "@mobalytics/shared": "file:../moba-shared/dist",
+     "@sentry/browser": "^6.5.1",
+     "@sentry/electron": "^3.0.7",
++    "@cliqz/adblocker-electron": "^1.25.1",
```

Comment:
Just checking this because I dont want to be releasing a version of mobaltyics they could get annoyed at.

I am fine with people modifying this themselves or adding it but I don't think releasing compiled versions with this to snap is really fair to them even if the ads are annoying.

If I get the time at the beginning of next year I may try to look to make an open-source multi-platform tool because it would be fun to try to make something that analyses the matches from the riot API.

Reply:
yeah thats fine